release.go

```diff
@@ -45,14 +45,15 @@ func NewReleaseID() ReleaseID {
 
 // Release describes a release
 type Release struct {
-	ID        ReleaseID            `json:"id"`
-	CreatedAt time.Time            `json:"createdAt"`
-	StartedAt time.Time            `json:"startedAt"`
-	EndedAt   time.Time            `json:"endedAt"`
-	Done      bool                 `json:"done"`
-	Priority  int                  `json:"priority"`
-	Status    ServiceReleaseStatus `json:"status"`
-	Log       []string             `json:"log"`
+	ID         ReleaseID            `json:"id"`
+	InstanceID InstanceID           `json:"instanceID"`
```

Comment:
Still needed? I think it was here so it could be fished out and used to log the event, but that doesn't happen any more.